docs/book/how-to/customize-docker-builds/how-to-reuse-builds.md

```diff
@@ -1,6 +1,37 @@
-# Use code repositories to speed up Docker build times
+---
+description: >
+  Learn how to reuse builds to speed up your pipeline runs.
+---
 
-While reusing Docker builds is useful, it can be limited. This is because specifying a custom build when running a pipeline will **not run the code on your client machine** but will use the code **included in the Docker images of the build**. As a consequence, even if you make local code changes, reusing a build will _always_ execute the code bundled in the Docker image, rather than the local code. Therefore, if you would like to reuse a Docker build AND make sure your local code changes are also downloaded into the image, you need to disconnect your code from the build.
+# How to reuse builds
+
+## What is a build?
+
+A pipeline build is an encapsulation of a pipeline and the stack it was run on. It contains the Docker images that were built for the pipeline with all the requirements from the stack, integrations and the user. Optionally, it also contains the pipeline code.
+
+When you run a pipeline, ZenML will check if a build with the same pipeline and stack exists. If it does, it will reuse that build. If it doesn't, ZenML will create a new build.
+
+You can list all the builds for a pipeline using the CLI:
+
+```bash
+zenml pipeline builds list --pipeline_id='startswith:ab53ca'
+```
+
+You can also create a build manually using the CLI:
+
+```bash
+zenml pipeline build [OPTIONS] SOURCE
```

Comment:
this is quite ambiguous without an example

Reply:
i agree. i've updated it with an example